media_driver/linux/common/ddi/media_libva.cpp

```diff
@@ -2208,6 +2208,7 @@ static VAStatus DdiMedia_DestroySurfaces (
         }
 
         DdiDecode_UnRegisterRTSurfaces(ctx, pSurface);
+        DdiEncode_UnRegisterRTSurfaces(ctx, pSurface);
```

Comment:
Can the function of DdiDecode/Encode_UnRegisterRTSurface be merged into one function? (DdiBase_UnRegiserRTSurface)
Otherwise it needs to call it twice when one surface is removed.